src/field_system.c

```diff
@@ -307,8 +307,8 @@ BgConfig *FieldSystem_GetBgConfigPtr(FieldSystem *fieldSystem) {
     return fieldSystem->bgConfig;
 }
 
-SaveData *FieldSystem_GetSaveDataPtr(FieldSystem *fieldSystem) {
-    return fieldSystem->savedata;
+SaveData *FieldSystem_GetSaveData(FieldSystem *fieldSystem) {
```

Comment:
I wonder if it's a good idea to have `Ptr` as implicit when other `Get` functions may return a struct and not a pointer.

Reply:
I litearlly know zero get functions that return a struct and not a pointer to one